frontend/src/pages/projects/screens/projects/ManageProjectModal.tsx

```diff
@@ -86,8 +86,6 @@ const ManageProjectModal: React.FC<ManageProjectModalProps> = ({
         </Button>,
       ]}
     >
-      <Stack hasGutter>
```

Comment:
Don't delete this -- this is a layout component -- we need to keep it for spacing.